docs/api.md

```diff
@@ -128,7 +128,7 @@ so helpers, etc work as expected.
 
 ## Configuration
 
-### `.capture_compatibility_patch_enabled` → [String]
```

Comment:
It looks like we lost our return types?

Reply:
That was intentional. We meant to exclude them for config docs, but the exclusion code was broken 😄